src/modules/Popup/Popup.js

```diff
@@ -76,6 +76,9 @@ export default class Popup extends Component {
     /** Horizontal offset in pixels to be applied to the popup */
     offset: PropTypes.number,
 
+    /** Vertical offset in pixels to be applied to the popup */
+    verticalOffset: PropTypes.number,
```

Comment:
How about allowing an X and Y offset in the current offset prop instead of adding a new prop? 

```jsx
offset={[5, 10]}

// vs

offset={5}
verticalOffset={10} 
```

We could still allow a single value to apply a horizontal offset only,so it is backwards compatible.